### PR TITLE
shadowsocks-libev: bump version to 2.5.0

### PR DESCRIPTION
**Pull request by aa65535**
Maintainer: @aa65535
Compile tested: x86/generic, VirtualBox, trunk
Run tested: x86, VirtualBox, trunk

Description: bump version to 2.5.0

Signed-off-by: Jian Chang <aa65535@live.com>

### CODE REVIEW

**Comment by yousong** [comment]
Version bump is not the only things changed.  A few words explaining other changes are needed in the commit message, especially about the removal of current uci related code.

File: net/shadowsocks-libev/Makefile [hunk 1]
```diff
@@ -8,13 +8,13 @@
...
 PKG_NAME:=shadowsocks-libev
-PKG_VERSION:=2.2.3
+PKG_VERSION:=2.5.0
 PKG_RELEASE:=1
 
 PKG_SOURCE_PROTO:=git
 PKG_SOURCE_URL:=https://github.com/shadowsocks/shadowsocks-libev.git
 PKG_SOURCE_SUBDIR:=$(PKG_NAME)-$(PKG_VERSION)-$(PKG_RELEASE)
-PKG_SOURCE_VERSION:=2b1eef11973de3f7380401fd20f937e84bc2b756
+PKG_SOURCE_VERSION:=983c1dcac0e845f68616bf55eea3cc8d5aa711df
```
Comment (yousong) [inline net/shadowsocks-libev/Makefile]: This commit id is not the tip of 2.5.0 tag.

**Comment by aa65535** [comment]
The uci related code  is not necessary, it has been moved to the [luci-app-shadowsocks-libev](https://github.com/openwrt/luci/pull/789).

**Comment by yousong** [comment]
That's something that needs to be part of the git history log.  The change is a major one because maintaining backward compatibility is deemed very important since years ago I started playing with OpenWrt and I don't think the rule has changed in this aspect.

Apart from the packaging method and package contents, UCI config is surely some abstraction we leverage upon to maintain stability across versions and releases and people benefit from this all these years.  The problem here is that users now depend on a previously optional LuCI app to make their old good uci config work again!

**Pull request closed without merging**